torch/multiprocessing/queue.py

```diff
@@ -22,7 +22,9 @@ def recv(self):
         return pickle.loads(buf)
 
     def __getattr__(self, name):
-        return getattr(self.conn, name)
+        if 'conn' in self.__dict__:
+            return getattr(self.conn, name)
+        raise AttributeError('no attribute: ' + name)
```

Comment:
Not sure what changed here. If conn is not in the dict it will give you an attribute error anyway

Reply:
This one is to avoid the calling of `__getattr__` inside this function, since the previous one will generate `max recursion depth exceeded`.

Reply:
Right, makes sense. Can you please change it to mimic the native error message better (like in [`nn.Module`](https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/module.py#L397-L398))?

Reply:
It is not necessary to add line 27.  Because the calling of `getattr` will generate recursion, we can call  `__getattr__` or test existence of 'conn' to avoid recursion.